pocs/utils/logger.py

```diff
@@ -87,11 +70,11 @@ def get_root_logger(profile='panoptes', log_config=None):
         full_log_fname = '{}/{}-{}.log'.format(log_dir, log_fname, handler)
         log_config['handlers'][handler].setdefault('filename', full_log_fname)
 
-        # Setup the TimeedRotatingFileHandler for middle of day
+        # Setup the TimedRotatingFileHandler for middle of day
```

Comment:
A TODO for the future: remove explicit reference to TimedRotatingFileHandler, and instead check for atTime being a string in the log_config, and if so parsing it with some flexibility regarding the time format.

Reply:
Not sure I understand what that would get us but let's discuss in a separate Issue.